sync/src/relayer/mod.rs

```diff
@@ -711,6 +715,12 @@ impl CKBProtocolHandler for Relayer {
             }
             ASK_FOR_TXS_TOKEN => self.ask_for_txs(nc.as_ref()),
             TX_HASHES_TOKEN => self.send_bulk_of_tx_hashes(nc.as_ref()),
+            SEARCH_ORPHAN_POOL_TOKEN => tokio::task::block_in_place(|| {
+                self.shared.try_search_orphan_pool(
+                    &self.chain,
+                    &self.shared.active_chain().tip_header().hash(),
+                )
+            }),
```

Comment:
https://github.com/nervosnetwork/p2p/blob/master/src/protocol_handle_stream.rs#L197
p2p already issuing `notify` handle as a blocking call, so I think  `tokio::task::block_in_place(` is unnecessary, also the `TX_PROPOSAL_TOKEN` handle.

Reply:
https://github.com/nervosnetwork/ckb/blob/develop/ckb-bin/src/subcommand/run.rs#L91
For performance and control, p2p blocking is disabled at the ckb level and it is controlled by itself